### PR TITLE
Update homepage

### PR DESCRIPTION
**Pull request by marksiemers**

### CODE REVIEW

**Comment by marksiemers** [comment]
Something funky is going on with the start count. It counts up correctly, then when I scroll up and back down again, it shows 575 - not sure what is causing that.

**Comment by elorest** [comment]
I wish.

**Comment by marksiemers** [comment]
If we can get a version of this approved and deployed before this Thursday, that will be helpful for my presentation.

**Comment by faustinoaq** [comment]
> Something funky is going on with the star count. It counts up correctly, then when I scroll up and back down again, it shows 575 - not sure what is causing that.

Yeah, I noticed that, the counter plugin comes with TheSaaS theme, What about removing counting section, or at least making it static?

File: src/views/home/index.slang [hunk 1]
```diff
@@ -114,19 +113,19 @@ main.main-content
           | Contributing to Amber can be a rewarding way to learn, teach, and build experience in just about any skill you can imagine. Amber is a community effort and we want You to be part of ours.
       .flexbox.flex-grow-all.text-center.mx-auto.amber-stats
```
Comment (faustinoaq) [inline src/views/home/index.slang]: Seems that counters, aren't working well (constantly outdated and sometimes wrong count)

What about removing this section or make it just static?

Maybe we can add real-time counters in the future 😅

**Comment by marksiemers** [comment]
@faustinoaq - I made the counters static and merged in the changes from master.

**Pull request merged**